src/Jsonize.Test/Program.cs

```diff
@@ -23,6 +25,18 @@ public static void Main(string[] args)
             Console.ReadLine();
         }
 
+        [SetUp] 
```

Comment:
[SetUp] and [TearDown] run before and after every test. You can read more about NUnit attributes [here](https://github.com/nunit/docs/wiki/Attributes). :)